internal/dbmodel/sql/postgres/1_13.sql

```diff
@@ -0,0 +1,8 @@
+-- 1_13.sql is a migration that introduces the concept of roles to JIMM.
+CREATE TABLE IF NOT EXISTS roles (
+   id BIGSERIAL PRIMARY KEY,
+   created_at TIMESTAMP WITH TIME ZONE,
```

Comment:
general question: why do we need to store the timestamp with timezone?

Reply:
I think may be a gorm thing, but I'm not 100%.